packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -105,3 +118,72 @@ export const getActivityList = createSelector(
     return visibleActivityList;
   }
 );
+
+/**
+ * Determines if an activity object is a visible activity
+ * @param {object} activity
+ * @returns {bool}
+ */
+function isActivityVisible(activity) {
+  if (!VISIBLE_ACTIVITY_VERBS.hasOwnProperty(activity.verb)) {
+    return false;
+  }
+  const verb = VISIBLE_ACTIVITY_VERBS[activity.verb];
+  if (verb.objectTypes) {
+    if (verb.objectTypes.indexOf(activity.object.objectType) === -1) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
Yea, we were doing something similar to this in a few different places in the code base.